src/main/java/pl/h2security/registration/exceptions/EmailAlreadyExistException.java

```diff
@@ -0,0 +1,7 @@
+package pl.h2security.registration.exceptions;
+
+public class EmailAlreadyExistException extends RuntimeException {
```

Comment:
Good job with extending Runtime. I would change the exception name to: EmailAlreadyUsedException